### PR TITLE
Organization endpoints for the site

### PR DESCRIPTION
**Pull request by ryanpbrewster**
Almost a direct copy from the Mobile controllers

@andrewconner @leogrim 

Take 2, let's see how it goes.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationMembershipController.scala [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              case Left(failure) => failure.asErrorResponse
+              case Right(responses) =>
+                val removals = responses.keys.map(r => externalIdMap(r.targetId))
+                Ok(Json.obj("removals" -> removals))
+            }
+        }
+    }
+  }
+
+  def leaveOrganization(pubId: PublicId[Organization]) = UserAction { request =>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationMembershipController.scala:99]: `OrganizationAction`?

**Comment by andrewconner** [comment]
Love seeing the tests. Looks good, we'll iterate.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationMembershipController.scala (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/OrganizationMembershipController.scala:99]: :sheep: Shucks.

**Comment by camhashemi** [comment]
@ryanpbrewster looks like your IDE re-formatted shoeboxService.routes. I just learned that even if this doesn't have any effect on production, it masks the git blame for those lines. May want to consider reverting that change.

**Comment by ryanpbrewster** [comment]
Damn, nice catch. Github hid that from me.

**Pull request merged**